projects/reservoir/fv3/era_process.py

```diff
@@ -37,18 +62,34 @@ def main(args):
     setup_fregrid()
     for var in variables:
         check_time_steps_complete(path, var)
-        weekdays = ["mon", "tue", "wed", "thu", "fri", "sat", "sun"]
-        for i, day in enumerate(weekdays):
-            merge_into_file(path, var, i, day)
-            logger.info(f"{day} is merged")
-            regrid_to_cubed_sphere(path, var, day)
-            logger.info(f"{day} is regridded")
-            interpolated = interpolate_nans(var, day)
-            logger.info(f"{day} is interpolated")
-            masked = mask(interpolated)
-            logger.info(f"{day} is masked")
-            masked.to_netcdf(args.out_path + var + "_" + day + ".nc")
-            logger.info(f"{day} is done")
+        merge_into_weekly_file(path, var, args)
+        logger.info(f"{var} is merged into weekly file")
+        regrid_to_cubed_sphere(path, var)
+        logger.info(f"{var} is regridded")
+        interpolated = interpolate_nans(var)
+        interpolated.to_netcdf("interpolated.nc")
+        logger.info(f"{var} is interpolated")
+        masked = mask(interpolated)
+        masked.to_netcdf("temp.nc")
+        logger.info(f"{var} is masked")
+        if var in ["temp2m", "u_wind", "v_wind"]:
+            shifted = shift_variable(masked)
```

Comment:
Can just call `xr.DataArray.shift` directly here and remove the `shift_variable` function.